authnz-theme/src/keycloak-theme/login/Template.tsx

```diff
@@ -130,6 +160,252 @@ export default function Template(props: TemplateProps<KcContext, I18n>) {
 		return null;
 	}
 
+	if (realm.internationalizationEnabled) {
+		assert(locale !== undefined);
+	}
+
+	const localizationOptions =
+		locale?.supported.map(locale => ({
+			value: locale.languageTag,
+			label: labelBySupportedLanguageTag[locale.languageTag],
+		})) ?? [];
+	const currentLocalizationOption = localizationOptions.find(
+		option => option.value === currentLanguageTag,
+	);
+
+	const onClickLightTheme = () => setTheme("light");
+	const onClickDarkTheme = () => setTheme("dark");
+	const onClickSystemTheme = () => setTheme("system");
+	const onChangeLocale = (next: ComboboxOption | null) => {
+		if (!next) {
+			return;
+		}
+
+		// TODO: <a> is more appropriate
+		location.href = getChangeLocaleUrl(next.value.toString());
+	};
+
+	const PageHeader = () => {
```

Comment:
<img width="1097" alt="image" src="https://github.com/user-attachments/assets/cd2d882c-9f5c-42c3-8a7e-d7eb0f739528">

the ternary is:
`!(auth !== undefined && auth.showUsername && !auth.showResetCredentials) = (auth === undefined && !auth.showUsername && auth.showResetCredentials)`

which is equivalent to handling two cases:
- auth.showUsername
- !auth.showUsername

Reply:
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/8535500d-7323-427c-a1a3-97fc1e8bd8be">